tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/ElasticSearchViewsSpec.scala

```diff
@@ -291,10 +291,10 @@ class ElasticSearchViewsSpec extends BaseSpec with EitherValuable with CirceEq {
         response.status shouldEqual StatusCodes.OK
         val expected = jsonContentOf(
           "/kg/views/statistics.json",
-          "total"     -> "14",
-          "processed" -> "14",
+          "total"     -> "5",
+          "processed" -> "5",
```

Comment:
I find it really weird that these changed. If this value could be anything, is it really useful?

Reply:
I see these statistics as "what delta has to work on" as such if we are now filtering types in the db then this is not something processed by delta anymore. I suppose the question is the utility of the test?

Reply:
I would question the utility of the statistic itself, but yes the test especially

Reply:
What I mean is: what is the utility of saying that I processed 14 things, when actually I didn't need to process 9 of them? Surely the only meaningful measure is of things that were actually affected

Reply:
To me when a method's efficiency is improved, if that causes a test to break then the test was not good

Reply:
Discarded to me was supposed to let the caller know that X elements have been dropped due to the filters they have specified in the view value. Just saying `processed: 5` doesn't do that. I suppose though with this change it could be confusing as one filter (the type filter) will not be part of the discarded anymore.

Reply:
We can discuss the statistics endpoint because its meaning has evolved with `1.8` and again with `1.9` and see what we can do (maybe just better document it first).